frontend/src/layout/navigation-3000/sidepanel/panels/SidePanelSupport.tsx

```diff
@@ -309,7 +310,10 @@ export const SidePanelSupport = (): JSX.Element => {
                                 </ul>
                             </Section>
 
-                            {!hasAvailableFeature(AvailableFeature.EMAIL_SUPPORT) ? (
+                            {!(
+                                hasAvailableFeature(AvailableFeature.EMAIL_SUPPORT) ||
+                                window.location.href.includes('/billing')
```

Comment:
This is confusing now. I'd switch up the cases so that it's looking for the `true` eval first